### ISSUE TITLE
[RFE] OpenSC support

### ISSUE THREAD

**Issue by jcpunk** [opendnssec/SoftHSMv2#440]
It would be amazing if SoftHSM could interface with OpenSC[1].  This would help folks looking to use both SoftHSM and a hardware token as well as extend the available query tools.

[1] https://github.com/OpenSC/OpenSC/wiki/Adding-a-new-card-driver

**Comment by jcpunk** [comment]
Currently OpenSC doesn't see the SoftHSM "reader" on my system.  I've really got two goals:

* move from proprietary tokens
* maintain most of my existing workflow

We've got a number of automation bits in place that check our tokens using OpenSC.  I'd prefer to transition just the tokens if possible.  

We've got some closed source applications that use OpenSC to locate the HSM.  Replacing them is on the list, but....

In theory, far down the road, GPG integration might be possible.  Right now GPG has limited support for OpenSC[1], getting support for SoftHSM in GPG would be really handy for my site.  But that is a challenge for another day.

[1] https://www.gnupg.org/documentation/manuals/gnupg/SmartCard_002dHSM.html
